autoinstall/Ubuntu/Server/user-data.j2

```diff
@@ -46,7 +46,7 @@ autoinstall:
     geoip: true
   packages:
     - sg3-utils
-{% if ubuntu_version is defined and ubuntu_version is version('22.04', '>=') %}
+{% if ubuntu_version is defined and ubuntu_version >= '22.04' and ubuntu_version != '25.04' %}
```

Comment:
Please keep using `is version` to compare versions.

Reply:
Change it. Thanks